deep_quoridor/src/agents/__init__.py

```diff
@@ -18,8 +18,8 @@ def _friendly_name(class_name: str):
         return class_name.replace("Agent", "").lower()
 
     @staticmethod
-    def create(friendly_name: str) -> "Agent":
-        return Agent.agents[friendly_name]()
+    def create(friendly_name: str, **kwargs) -> "Agent":
```

Comment:
some agents may need some environment info to be created. The base class should allow for parameters and the Arena should make sure to pass the info down.